agent/util-scripts/pbench-tool-meister-start

```diff
@@ -140,12 +140,11 @@ from pbench.agent.utils import (
 _TDS_STARTUP_TIMEOUT = 60
 
 
-class ReturnCode:
+class ReturnCode(BaseReturnCode):
```

Comment:
> I'm unclear on how we're going to avoid name collisions on values

Good question, I'll investigate; one possibility is to employ a unit test to compare the base class codes against the sub-classes to ensure no overlap.

> how we avoid unintentional value collisions

The way the `ReturnCode` "kill" works is it shifts the codes by 1,000.

Reply:
OK, so the `KILL_` values are a different "type".  (One wonders if they shouldn't be in their own `IntEnum`.)

Reply:
Tracking in #2235.